castle/migrations/0029_challenge.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
So, yeah, technically they _are_ both important and temporary, in that they get used only once but they're essential to the function of the site.